[DA] TapViewButton 눌렀을 때 여러번 호출되는 문제 수정

kimkyunghun3 opened the pull request:
# 개요
- 🔗  이슈링크 : #180

## 작업 내용
- TapViewButton 눌렀을 시 중복호출되는 현상 해결


### 스크린샷
![해결?](https://user-images.githubusercontent.com/52434820/186455609-d26f812a-1c5e-4c6f-a733-1a4db33dff0a.gif)

kimkyunghun3 committed:
:recycle: TapView 누르시 여러번 호출되는 문제 해결

# :tada: = 프로젝트 시작 (#)
# :sparkles: = create 파일 생성 시 (#)
# :bulb: = update 코드 및 UI 업데이트, 기능 구현 시 (#)
# :rocket: = release 배포 시 (#)
# :recycle: = refactor 리팩토링 시 (기능은 같은데 코드가 달라지는경우) (#)
# :memo: = 문서 작성 시 (#)
# :art: = 디자인 시스템 관련 내용 (Color / Asset) (#)
# :hammer: = 테스트 코드 작성 (#)
################

# 커밋 타입 : 커밋 제목 (계정명/저장소명#이슈번호) 형식으로 작성하며 제목은 최대 50글자 정도로만 입력
# 제목을 아랫줄에 작성, 제목 끝에 마침표 금지, 무엇을 했는지 명확하게 작성

szzang-kr reviewed (dismissed):
https://velog.io/@dvhuni/UITextField에서-rx.text로-변경된-텍스트를-감지하기

`.distinctUntilChanged()`에 대한 설명이 있습니다. 참고하시면 좋아요~!!

kimkyunghun3 committed:
:recycle: 응모화면 notice 내용 변경

# :tada: = 프로젝트 시작 (#)
# :sparkles: = create 파일 생성 시 (#)
# :bulb: = update 코드 및 UI 업데이트, 기능 구현 시 (#)
# :rocket: = release 배포 시 (#)
# :recycle: = refactor 리팩토링 시 (기능은 같은데 코드가 달라지는경우) (#)
# :memo: = 문서 작성 시 (#)
# :art: = 디자인 시스템 관련 내용 (Color / Asset) (#)
# :hammer: = 테스트 코드 작성 (#)
################

# 커밋 타입 : 커밋 제목 (계정명/저장소명#이슈번호) 형식으로 작성하며 제목은 최대 50글자 정도로만 입력
# 제목을 아랫줄에 작성, 제목 끝에 마침표 금지, 무엇을 했는지 명확하게 작성
